modules/node_groups/launch_template.tf

```diff
@@ -24,6 +24,20 @@ data "cloudinit_config" "workers_userdata" {
   }
 }
 
+data "template_file" "bottlerocket_workers_userdata" {
```

Comment:
template_file is a deprecated provider.

It is better to use the templatefile function.

Reply:
thansks @ulm0 ! So fast ;)

I got an issue with this provider a few days ago as it is not available for arm platforms and won't be (deprecation notice) :D

Reply:
just did a minor fix, should be good to go now @jlamande